cypress/e2e/signIn.cy.js

```diff
@@ -1,25 +1,30 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
 
+import SignUpPageObject from '../support/pages/signUp.pageObject';
 import SignInPageObject from '../support/pages/signIn.pageObject';
 import HomePageObject from '../support/pages/home.pageObject';
 
+const signUpPage = new SignUpPageObject();
 const signInPage = new SignInPageObject();
 const homePage = new HomePageObject();
 
 describe('Sign In page', () => {
   let user;
 
-  before(() => {
+  beforeEach(() => {
     cy.task('db:clear');
```

Comment:
Clearing the database in the `beforeEach` hook means that the database is reset before each test. This might not be necessary if the tests are independent and could slow down the test suite. Consider whether this is required for each test or if it can be optimized.